src/pages/MetricDetail.svelte

```diff
@@ -92,7 +93,7 @@
         {#if metric.origin !== params.app}
           <a href={getMetricSearchURL(params.app, metric.origin)}
             ><Label
-              text={metric.origin}
+              text={getLibraryName(metric)}
               description={getLibraryDescription("metric", metric.origin)}
```

Comment:
The amount of repetition in this PR indicates we might want to create a higher-level abstraction for displaying tags, but let's leave that for later.